pages/Nvidia/_index.md

```diff
@@ -8,6 +8,20 @@ Every card seems to be random, and might work perfectly, or not work at all.
 
 Nevertheless, it's recommended to at least try this tutorial.
 
+## Hyprland Nvidia Patch (Arch only) (Unofficial)
+```sh
+hyprland-nvidia (AUR)
+hyprland-nvidia-git (AUR)
+```
+{{< hint type=warning >}}
+Hyprland Nvidia Patch is **NOT** an official patch and is not maintained by us.
+If you have any concerns (updates, broken pkgbuild, etc), you should contact the maintainer.
+{{< /hint >}}
+
+{{< hint type=tip >}}
+Hyprland Nvidia Patch at the end, is simply Hyprland with some modifications, if this package doesn't work for you or you have any issues (eg. flickering), keep reading below, you may still have a chance.
```

Comment:
Mention the initramfs & kernel parameters are still required. These patches fix wonky behavior (flickers etc) but still require the below